controllers/cluster_controller.go

```diff
@@ -448,6 +482,24 @@ func (r *FoundationDBClusterReconciler) getStatusFromClusterOrDummyStatus(logger
 		}, nil
 	}
 
+	/*Get the status of all the pods in the cluster.
+	 *Until all the pods are up it will keep cluster in config missing state and ask reconciler to spawn required pods/pvc, etc.
+	 *Once all the pods are up and running it will force the cluster to reconcile
+	 */
+	podsStatus := r.getAllPodStatus(ctx, logger, cluster)
+	if !podsStatus &&
+		(cluster.Status.Generations.NeedsConfigurationChange == 1 ||
+			cluster.Status.Generations.DatabaseUnavailable == 1 ||
+			cluster.Status.Generations.HasUnhealthyProcess == 1) {
```

Comment:
```suggestion
		(cluster.Status.Generations.NeedsConfigurationChange > 0 ||
			cluster.Status.Generations.DatabaseUnavailable > 0 ||
			cluster.Status.Generations.HasUnhealthyProcess > 0) {
```

Otherwise this check will only work for the first generation (initial state).